ops/reth-cross-test/verification/stress_runner.py

```
@@ -0,0 +1,121 @@
+        self._stop = True
+        for t in threads:
+            t.join(timeout=10)
+
+    def _worker(self, worker_id: int, contract_addr: str = None):
+        """Worker thread that sends transactions."""
+        w3 = Web3(Web3.HTTPProvider(self.rpc_url, request_kwargs={"timeout": 30}))
+        account = self._accounts[worker_id % len(self._accounts)]
+
+        while not self._stop:
+            try:
+                roll = random.random()
```

Ruff (0.15.7)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)